eta/core/video.py

```diff
@@ -1530,7 +1530,7 @@ def sample_select_frames(video_path, frames, output_patt=None, fast=False):
         return imgs
 
 
-def sample_first_frames(arg, k, size=None):
+def sample_first_frames(arg, k, stride=2, size=None):
```

Comment:
you're changing the default behavior! should be `stride=1`

Reply:
need to document new parameter in docstring

Reply:
Oh right, my bad.